src/main/kotlin/bd/clinic/modules/address/Address.kt

```diff
@@ -0,0 +1,11 @@
+package bd.clinic.modules.address
+
+import javax.persistence.Embeddable
+
+@Embeddable
+class Address(
+        var streetName: String,
```

Comment:
Zrobiłem `var`, bo pewnie będzie możliwość zmiany adresu

Reply:
Choć w sumie jak jest `val` to i tak można zmienić te pola bo tak naprawdę się wszystko nadpisuje przy `PUT` (poza `id` ofc)